Makefile

```diff
@@ -6,7 +6,7 @@
 	build
 
 SOURCES := $(shell find . -name '*.go' -not -path "*/vendor/*")
-IMG ?= quay.io/updateservice/updateservice-operator:latest
+IMG ?= quay.io/updateservice/update-service-operator:latest
```

Comment:
do we actually control this Quay namespace?  I prefer [the SDK's scaffolded `IMG ?= controller:latest`][1], which is more obviously a placeholder.

[1]: https://github.com/openshift/cincinnati-operator/commit/cfc695634ec34a3c602f34176fea7215fe3e73a3#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R15

Reply:
I'm not sure how this works either but there's an existing `cincinnati` organization, quay.io/cincinnati, containing repos cincinnati and cincinnati-operator. So I thought we would have the same but with update-service.

Reply:
Also, whatever we decide, this default is also used other places, most notably in `config/manager/manager.yaml`, which then allows use of `kustomize`,  or `hack/deploy.sh`, to change it to a chosen image.

Reply:
Right now I have used `IMG ?= controller:latest` in all the places as we do not have `quay.io/updateservice/update-service-operator` in quay. Once we figure out what namespace we want to use for the CI we can change these back.

Reply:
Resolved in https://github.com/openshift/cincinnati-operator/commit/f99ff7df60ccd1ffae42fcb1a4cd8b6f3d78027f